### PR TITLE
Fix 32-bit build

### PR DESCRIPTION
**Pull request by ifratric**
Thanks for the great library! :-)

If a 32-bit build is created, it will fail when attempting to get and subsequently parse the trace. This PR fixes this in the following ways:

 - Since HANDLE is defined as a void *, its size is going to be different on 32 and 64, resulting in different structure alignment between platforms. To resolve this, I added explicit padding before ProcessHandle in GetProcessIptTraceSize and GetProcessIptTrace requests.

 - In IPT_TRACE_HEADER ThreadId was declared as HANDLE, however a thread ID is usually a DWORD. This means there needs to be an additiona 4 byte field following it (its purpose unknown to me), so that the offset of TimingSettings can be brought up to 8.

### CODE REVIEW

**Comment by ionescu007** [comment]
Does IPT.SYS actually work/exist on 32-bit systems at all?

File: inc/libiptnt.h [hunk 1]
```diff
@@ -88,7 +88,8 @@ typedef struct _IPT_TRACE_DATA
 
 typedef struct _IPT_TRACE_HEADER
 {
-    HANDLE ThreadId;
+    DWORD ThreadId;
```
Comment (ionescu007) [inline inc/libiptnt.h]: DWORD is not an NT type, ULONG is, FYI.

**Comment by ifratric** [comment]
I have no idea about fully 32-bit systems, but in some cases (e.g. mine :-) it might be useful to have a 32-bit build running on a 64-bit system, depending on what else is running in the same process. And it seems a straightforward change. :-)

I changed DWORD to ULONG in the NT library.

File: inc/libiptnt.h [hunk 2]
```diff
@@ -88,8 +88,8 @@ typedef struct _IPT_TRACE_DATA
 
 typedef struct _IPT_TRACE_HEADER
 {
-    DWORD ThreadId;
-    DWORD Unknown;
+    ULONG ThreadId;
```
Comment (ionescu007) [inline inc/libiptnt.h]: OK, I verified with x86 -- there's no other field, this should just be ULONG64 in NT, and DWORD64 in Win32. Please make the change (and necessary casts where needed).

File: inc/ipt.h [hunk 3]
```diff
@@ -41,11 +41,13 @@ typedef struct _IPT_INPUT_BUFFER
         struct
         {
             USHORT TraceVersion;
+            BYTE padding[6];
             HANDLE ProcessHandle;
         } GetProcessIptTraceSize;
         struct
         {
             USHORT TraceVersion;
+            BYTE padding[6];
```
Comment (ionescu007) [inline inc/ipt.h]: Please replace all instances of HANDLE to ULONG64 and remove the BYTE padding -- thanks!

**Comment by ionescu007** [comment]
Thanks for figuring out these issues with x86/Wow64. I supplied comments on the appropriate fixes -- I'll merge once that's done, thx!

**Review by ionescu007 — APPROVED** [review]
LGTM -- thanks again

**Pull request merged**